docs/layouts/shortcodes/generated/pulsar_consumer_configuration.html

```diff
@@ -0,0 +1,180 @@
+            <td style="word-wrap: break-word;">0</td>
+            <td>Long</td>
+            <td>Set the timeout (in ms) for unacknowledged messages, truncated to the nearest millisecond. The timeout needs to be greater than 1 second.<br />By default, the acknowledge timeout is disabled and that means that messages delivered to a consumer will not be re-delivered unless the consumer crashes.<br />When enabling ack timeout, if a message is not acknowledged within the specified timeout it will be re-delivered to the consumer (possibly to a different consumer in case of a shared subscription).</td>
+        </tr>
+        <tr>
+            <td><h5>pulsar.consumer.acknowledgementsGroupTimeMicros</h5></td>
+            <td style="word-wrap: break-word;">100000</td>
+            <td>Long</td>
+            <td>Group a consumer acknowledgment for a specified time (in μs). By default, a consumer uses <code class="highlighter-rouge">100μs</code> grouping time to send out acknowledgments to a broker. Setting a group time of <code class="highlighter-rouge">0</code> sends out acknowledgments immediately. A longer ack group time is more efficient at the expense of a slight increase in message re-deliveries after a failure.</td>
+        </tr>
+        <tr>
+            <td><h5>pulsar.consumer.autoAckOldestChunkedMessageOnQueueFull</h5></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>Buffering large number of outstanding uncompleted chunked messages can create memory pressure and it can be guarded by providing this <code class="highlighter-rouge">pulsar.consumer.maxPendingChunkedMessage</code> threshold. Once consumer reaches this threshold, it drops the outstanding unchunked-messages by silently acking if <code class="highlighter-rouge">pulsar.consumer.autoAckOldestChunkedMessageOnQueueFull</code> is true else it marks them for redelivery.</td>
+        </tr>
+        <tr>
+            <td><h5>pulsar.consumer.autoUpdatePartitions</h5></td>
+            <td style="word-wrap: break-word;">true</td>
+            <td>Boolean</td>
+            <td>If <code class="highlighter-rouge">autoUpdatePartitions</code> is enabled, a consumer subscribes to partition increase automatically.<br />Note: this is only for partitioned consumers.</td>
+        </tr>
+        <tr>
+            <td><h5>pulsar.consumer.autoUpdatePartitionsIntervalSeconds</h5></td>
+            <td style="word-wrap: break-word;">60</td>
+            <td>Integer</td>
+            <td>Set the interval (in seconds) of updating partitions. This only works if autoUpdatePartitions is enabled.</td>
+        </tr>
+        <tr>
+            <td><h5>pulsar.consumer.consumerName</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>Consumer name is informative and it can be used to identify a particular consumer instance from the topic stats.</td>
```

Comment:
```suggestion
            <td>The consumer name is informative and it can be used to identify a particular consumer instance from the topic stats.</td>
```